chainerx_cc/chainerx/python/routines.cc

```diff
@@ -58,7 +58,10 @@ ArrayBodyPtr MakeArrayFromBuffer(py::buffer buffer, py::handle dtype, int64_t co
         throw ChainerxError{"offset must be non-negative and no greater than buffer length (", n_bytes, ")"};
     }
 
-    if (!internal::IsContiguous(Shape{info.shape}, Strides{info.strides}, info.itemsize)) {
+    Shape info_shape{info.shape.begin(), info.shape.end()};
+    Strides info_strides{info.strides.begin(), info.strides.end()};
+
+    if (!internal::IsContiguous(info_shape, info_strides, info.itemsize)) {
```

Comment:
I think you can inline `info_{shape,strides}` without giving them names (but use the same ctors) since they're not used afterwards.

Reply:
Yes you're right. I don't know why that didn't occur to me.